### DISCUSSION TITLE
Attribute "Use as filter", button purpose?

### DISCUSSION THREAD

**Discussion by jonathan-franzen** [saleor/saleor#14127] [Q&A] [answered]
Hi, when creating attributes in admin, there is a selection to the right "use as filter" Ive been truing enabling/ disabling this back and forth, but I can't still figure out the purpose of this buuton. What does it actually to to the attribute?

<img width="1201" alt="Screenshot 2023-09-29 at 00 24 34" src="https://github.com/saleor/saleor/assets/91531812/e7012969-2b9d-4a42-bdc5-433887650612">

**Comment by timuric** [comment, marked as answer]
It is meant to signal to front end if this attribute should be public to users. It doesn't alter behavior of saleor it self
